server.go

```diff
@@ -230,6 +230,19 @@ func (s *Server) load() error {
 			}
 		}
 
+		// Open all shards.
```

Comment:
When the server starts up it just loads the shard metadata but doesn't actually open the underlying shard store. If you do a query before writing data to the shard after restart then it'll fail.